### PR TITLE
Feat 4466 deepgram language detection using node

### PR DESCRIPTION
**Pull request by Player256**
Fixes [appwrite/appwrite#4466](https://github.com/appwrite/appwrite/issues/4466)
Command Prompt 05-03-2023 19_22_05

### CODE REVIEW

**Comment by Player256** [comment]
@stnguyen90 can you give it a look

File: node/deepgram_language_detection/src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const { Deepgram } = require("@deepgram/sdk");
+
+module.exports = async (req, res) => {
+  const { fileUrl } = JSON.parse(req.payload);
+  const {apiKey} = JSON.parse(req.variables["DEEPGRAM_API_KEY"]);
```
Comment (stnguyen90) [inline node/deepgram_language_detection/src/index.js:5]: You shouldn't need to parse the variable like this 🧐

File: node/deepgram_language_detection/src/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  if (!apiKey) {
+    res.json({
+      success: false,
+      message: "API key is required.",
+    });
```
Comment (stnguyen90) [inline node/deepgram_language_detection/src/index.js:11]: make sure to `return` after this so execution doesn't continue

File: node/deepgram_language_detection/src/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  if (!fileUrl) {
+    res.json({
+      success: false,
+      message: "File Url is required.",
+    });
```
Comment (stnguyen90) [inline node/deepgram_language_detection/src/index.js:18]: make sure to `return` after this so execution doesn't continue

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

**Comment by stnguyen90** [comment]
@Player256, will you be able to address the comments I left?

**Pull request closed without merging**